### PR TITLE
New package: pdfpc-4.0.3

### PR DESCRIPTION
**Pull request by drkhsh**

### CODE REVIEW

File: srcpkgs/pdfpc/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'pdfpc'
+pkgname=pdfpc
+version=4.0.3
+revision=1
+build_style=cmake
+wrksrc=${pkgname}-${version}-1
+makedepends="pkg-config cmake vala vala-devel gtk+3-devel poppler-glib-devel libgee08-devel gstreamer1-devel gst-plugins-base1-devel"
```
Comment (ebfe) [inline srcpkgs/pdfpc/template]: *-devel can stay in makedepends, the rest should go into hostmakedepends (probably fixes cross builds)

File: srcpkgs/pdfpc/template [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+depends="gtk+3 poppler-glib libgee08 gstreamer1 gst-plugins-base1-devel"
```
Comment (ebfe) [inline srcpkgs/pdfpc/template]: these should be automatically detected and likely can be removed.

File: srcpkgs/pdfpc/template [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+short_desc="A presenter console with multi-monitor support for PDF files"
+maintainer="Aaron Marcher <info@nulltime.net>"
+license="GPL-2"
+homepage="https://pdfpc.github.io/"
+distfiles="https://github.com/pdfpc/pdfpc/archive/v${version}-1.tar.gz>pdfpc-${version}.tar.gz"
+checksum=6ead2fc793cab952ee10a363552ce0391e696a63a4e262865d0579da55a6dac0
+
+post_install() {
+	vlicense LICENSE.txt
```
Comment (ebfe) [inline srcpkgs/pdfpc/template]: not necessary if license is GPL-2.

File: srcpkgs/pdfpc/template (hunk 1 above)
Comment (ebfe) [inline srcpkgs/pdfpc/template]: also can drop cmake it's implicit with `build_style=cmake`

File: srcpkgs/pdfpc/template (hunk 2 above)
Comment (Vaelatern) [inline srcpkgs/pdfpc/template]: https://travis-ci.org/voidlinux/void-packages/jobs/174885840#L951-L964 
Yes, can be removed.

**Comment by drkhsh** [comment]
i will have a look, thanks

**Comment by drkhsh** [comment]
@ebfe @Vaelatern i fixed everything, please have a look! :grin: :+1:

**Comment by pullmoll** [comment]
Please squash your commits into one using `git rebase -i HEAD~2`, select `s` for the `Fixed pdfpc` entry and `git push -f` afterwards.

For the future, if you make changes to an existing PR, just use `git commit --amend` and `git push -f` to avoid multiple commits for a single package.

**Comment by drkhsh** [comment]
@pullmoll done, sorry

**Comment by drkhsh** [comment]
so when will this be merged?

**Pull request merged**